.gitignore

```diff
@@ -26,14 +26,11 @@ hs_err_pid*
 
 /values/google_maps_api.xml
 /local.properties
-/.idea/caches
-/.idea/libraries
-/.idea/modules.xml
-/.idea/workspace.xml
-/.idea/navEditor.xml
-/.idea/assetWizardSettings.xml
+.idea
 .DS_Store
 /build
 /captures
 .externalNativeBuild
 .cxx
+
+library/a-star/build
```

Comment:
The file underneath, .gitignore.swp, what is that? I think u shouldnt add it to master

Reply:
its a temp file, happens when you commit and didn't close all your files, I've deleted it